README.md

```diff
@@ -124,7 +123,7 @@ html_path = vis.render(open_browser=True, read_only=False)
 **Visualize**
 
 ```py
-Visualize(data=None, names=None, read_only=False, api_key=None, style=None)
+Visualize(data=None, names=None, read_only=False, api_key=None, style=None, config_file=None, output_map=None)
```

Comment:
See below for discussion of whether `open_browser` should exist as a param to `__init__`, but if it does exist then it should be documented here